roles/local_storage/tasks/create_filesystem_on_device.yml

```diff
@@ -30,6 +30,18 @@
     # and when the device is unformatted.
     #
     - mount_point_status['rc'] != 0
-    - device_status['stdout'] != volume_type
+    - device_status['stdout'] != volume['type']
+  become: true
+
+- name: Label existing, unmounted ext file system.
```

Comment:
e2label works only for `ext` filesystems. But not for xfs, btrfs. Either this should be noted or fixed.
`e2label - Change the label on an ext2/ext3/ext4 filesystem`

Default filesystem on on RHEL 7, 8, 9 (and is probably also going to be on 10) is XFS.

`xfs_admin -L LOR-root /dev/sda1`?